xbmc/input/Action.cpp

```diff
@@ -75,7 +75,7 @@ CAction::CAction(int actionID, const std::string &name, const CKey &key):
     m_amount[i] = 0;
   m_repeat = key.GetRepeat();
   m_buttonCode = key.GetButtonCode();
-  m_unicode = 0;
+  m_unicode = key.GetUnicode();
```

Comment:
@garbear was there a special reason why unicode was set to 0?

Reply:
This code was c/p from the original code written 10 years ago. I suspect it's been wrong this whole time.

Reply:
Then better bring this in? ;-)

Reply:
Do it